index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Slide Show</title>
+		<style>
+		body{
+
+		}
+		.slideShow-container{
+			background-color: rgba(255,255,255,0.3);
+			position: absolute;
+			height: 95%;
+			width: 100%;
+			top: 10%;
+			left: 0%;
+		}
+		.slides-container{
+			position: absolute;
+			height: 100%;
+    		width: 100%;
+    		top: 0;
+    		left: 0;
+
+		}
+		.btn{
+			background-repeat: no-repeat;
+    		position: absolute;
+    		top: 0;
+    		height: 100%;
+    		width: 4%;
+    		cursor: pointer;
+    		opacity: 0.3;
+		}
+		.Option-bar{
+			background-color: rgba(0,0,0,0.5);
+			position: absolute;
+			top: 94%;
+			height: 6%;
+			width: 100%;
+		}
+		.play{
+			left: 75%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/circled-play.png);
+		}
+		.prev{
+			left: 65%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/back.png);
+		}
+		.next{
+			left: 85%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/forward.png);
+		}
+		.thumbnails{
+			left: 23.5%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/stack-of-photos.png);
+		}
+		.info{
+			left: 10%;
+			background-image: url(https://png.icons8.com/nolan/50/000000/plus-math.png);
+		}
+		.counter{
+    		position: absolute;
+    		top: 0;
+    		height: 100%;
+    		width: 10%;
+    		left: 45%;
+    		text-align: center;
+			margin: auto;
+			height: 100%;
+			font-size: 255%;
+		}
+		.imgs{
+			position: absolute;
+			height: 100%;
+			overflow: hidden;
+			visibility: hidden;
+			width: 100%
+		}
+		.imgs img{
+			position: absolute;
+			display: block;
+    		top: 0;
+    		left: 0;
+    		right: 0;
+    		bottom: 0;
+    		height: 100%;
+    		margin: auto;
+		}
+		.thumbnails-popUp{
+			display: table;
+			background-color: rgba(0,0,0,0.5);
+			height: 15%;
+			position: absolute;
+			width: 100%;
+			bottom: 6%;
+			visibility: hidden;
+		}
+		.title{
+		    height: 6%;
+		    position: absolute;
+		    top: 0%;
+		    background-color: rgba(255,255,255,69%);
+		    text-align: center;
+		    /* padding-top: 0%; */
+		    width: 100%;
+		    /* left: 50%; */
+		    /* transform: translate(-50%,0); */
+		}
+		.paragraph{
+		    height: 20%;
+		    position: relative;
+		    width: 47%;
+		    left: 1%;
+		    top: 71%;
+		    text-align: justify;
+		    background-color: rgba(100%, 100%, 100%, 42%);
+		    padding: 10px;
+		    display: none;
+		    /* transform: translate(-50%,0%); */
+		    /* margin: auto; */
+		    /* bottom: 1%; */
+		}
+		.thumbnails-imgs{
+			float: left;
+			width: 20%;
+		}
+		.thumbnails-imgs img{
+			width: 100%;
+			height:	189px;
+			opacity: 0.6;
+			cursor: pointer;
+		}
+
+		
+	</style>
+</head>
+<body>
+	<h2>SlideShow</h2>
```

Comment:
Every page should have a h1 for accessibility sake. Maybe you want to chage it to h1

Reply:
I didn't know, I'll keep it in mind from now on.